lib/kubernetes-deploy/kubectl.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'kubeclient'
+
+module KubernetesDeploy
```

Comment:
I agree with Katrina and I'd also prefer moving `kubectl` helpers in a separate PR.